package.json

```diff
@@ -12,21 +12,25 @@
     "test": "jest"
   },
   "dependencies": {
-    "babel-eslint": "8.2.6",
+    "@typescript-eslint/eslint-plugin": "4.12.0",
+    "@typescript-eslint/parser": "4.12.0",
+    "babel-eslint": "10.1.0",
     "codecov": "3.6.5",
-    "eslint": "5.4.0",
-    "eslint-config-prettier": "3.0.1",
-    "eslint-config-vazco": "4.0.0",
-    "eslint-plugin-babel": "5.1.0",
-    "eslint-plugin-prettier": "2.6.2",
-    "eslint-plugin-react": "7.11.1",
-    "eslint-plugin-vazco": "1.0.0",
+    "eslint": "7.17.0",
+    "eslint-config-prettier": "7.1.0",
+    "eslint-config-vazco": "6.1.0",
```

Comment:
How about upgrading `package-json.lock` to v2 and require npm@7 in the `engines` section? It'd let us get rid of all of these lines here.

Reply:
Sounds good to me, changed.